pkgs/os-specific/linux/libzbc/default.nix

```
@@ -0,0 +1,40 @@
+{
+  autoconf-archive,
+  autoreconfHook,
+  fetchFromGitHub,
+  gtk3,
+  lib,
+  libtool,
+  pkg-config,
+  stdenv,
+  guiSupport ? false,
+}:
+stdenv.mkDerivation rec {
+  pname = "libzbc";
+  version = "5.12.0";
+
+  src = fetchFromGitHub {
+    owner = "westerndigitalcorporation";
+    repo = "libzbc";
+    rev = "v${version}";
+    sha256 = "qI09dkMCwMym3j1ELrFDNbNB5hW/CzwmFmZhUNDXsfI=";
+  };
+
+  nativeBuildInputs = [
+    autoconf-archive # this can be removed with the next release
+    autoreconfHook
+    libtool
+  ];
+
+  buildInputs = lib.optionals guiSupport [gtk3 pkg-config];
```

```suggestion
  buildInputs = lib.optionals guiSupport [ gtk3 pkg-config ];
```
pkg-config goes to nativeBuildInputs.